_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,455 @@
+---
+id: 9bd38c
+name: Content has non-visual reference alternative
+rule_type: atomic
+description: |
+  This rule checks that when content is identified through a visual reference, there are also non-visual references identifying the same content.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.3.3: # Sensory Characteristics (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G96: # Providing textual identification of items that otherwise rely only on sensory information to be understood.
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS Styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Daniël Strik
+    - Jean-Yves Moyen
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [text node][] that is either [visible][] or [included in the accessibility tree][].
+
+## Expectation
+
+For each test target that includes at least one of the [visual reference words][], one of the following is true:
+
+- (**no indication**) the test target does not identify any [web content][] through the use of any [visual reference words][]; or
```

Comment:
Likely because this rings a bell with the way I use to number/name equations in mathematical texts. The bracket+bold is also a reminder in the running text that this refers to one of the condition (while bold could be used for emphasise in general).